src/main/java/io/stargate/sgv2/jsonapi/service/bridge/executor/QueryExecutor.java

```diff
@@ -106,6 +107,29 @@ private Uni<QueryOuterClass.ResultSet> queryBridge(QueryOuterClass.Query query)
             });
   }
 
+  /**
+   * Gets the schema for the provided namespace and collection name
+   *
+   * @param namespace
+   * @param collectionName
+   * @return
+   */
+  protected Uni<Optional<Schema.CqlTable>> getSchema(String namespace, String collectionName) {
+    Schema.DescribeKeyspaceQuery describeKeyspaceQuery =
+        Schema.DescribeKeyspaceQuery.newBuilder().setKeyspaceName(namespace).build();
+    final Uni<Schema.CqlKeyspaceDescribe> cqlKeyspaceDescribeUni =
+        stargateRequestInfo.getStargateBridge().describeKeyspace(describeKeyspaceQuery);
+    return cqlKeyspaceDescribeUni
+        .onItem()
+        .transform(
+            cqlKeyspaceDescribe -> {
+              Schema.CqlTable cqlTable = null;
+              return cqlKeyspaceDescribe.getTablesList().stream()
+                  .filter(table -> table.getName().equals(collectionName))
+                  .findFirst();
```

Comment:
What happens if no table matches name? Exception?

Reply:
If no metadata for the table the method will return Optional.empty(), which is handled in NamespaceCache.isVectorEnabledInternal